srcpkgs/brillo/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'brillo'
+pkgname=brillo
+version=1.2.0
+revision=1
+build_style=gnu-makefile
+wrksrc="$pkgname-v$version"
+make_build_args="dist"
+make_install_args="install-dist"
+makedepends="help2man"
```

Comment:
help2man is meant to be executed on the host system so move to `hostmakedepends`